<ui version="4.0" >
 <class>ArchiveOptionsClass</class>
 <widget class="QWidget" name="ArchiveOptionsClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>ArchiveOptions</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="grbMethod" >
     <property name="title" >
      <string>Archiving Method</string>
     </property>
     <layout class="QGridLayout" >
      <property name="leftMargin" >
       <number>5</number>
      </property>
      <property name="topMargin" >
       <number>5</number>
      </property>
      <property name="rightMargin" >
       <number>5</number>
      </property>
      <property name="bottomMargin" >
       <number>5</number>
      </property>
      <item row="0" column="0" >
       <widget class="QLabel" name="lblMethodLocal" >
        <property name="text" >
         <string>Local archiving method:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QComboBox" name="cmbMethodLocal" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="lblMethodAuto" >
        <property name="text" >
         <string>Auto archiving method:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QComboBox" name="cmbMethodAuto" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="lblMethodManual" >
        <property name="text" >
         <string>Manual archiving method:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QComboBox" name="cmbMethodManual" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2" >
       <widget class="QCheckBox" name="chbAutoSave" >
        <property name="text" >
         <string>Enable server-side auto archiving</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grbDefault" >
     <property name="title" >
      <string>Default Archiving Preferences</string>
     </property>
     <layout class="QGridLayout" >
      <property name="leftMargin" >
       <number>5</number>
      </property>
      <property name="topMargin" >
       <number>5</number>
      </property>
      <property name="rightMargin" >
       <number>5</number>
      </property>
      <property name="bottomMargin" >
       <number>5</number>
      </property>
      <item row="0" column="0" >
       <widget class="QLabel" name="lblModeOTR" >
        <property name="text" >
         <string>Off The Record Mode</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLabel" name="lblModeSave" >
        <property name="text" >
         <string>Save Mode:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QLabel" name="lblExpireTime" >
        <property name="text" >
         <string>Expire Time, days</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QComboBox" name="cmbModeOTR" />
      </item>
      <item row="1" column="1" >
       <widget class="QComboBox" name="cmbModeSave" />
      </item>
      <item row="1" column="2" >
       <widget class="QComboBox" name="cmbExpireTime" >
        <property name="editable" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grbIndividual" >
     <property name="title" >
      <string>Individual Archiving Preferences</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="leftMargin" >
       <number>5</number>
      </property>
      <property name="topMargin" >
       <number>5</number>
      </property>
      <property name="rightMargin" >
       <number>5</number>
      </property>
      <property name="bottomMargin" >
       <number>5</number>
      </property>
      <item>
       <widget class="QTableWidget" name="tbwItemPrefs" >
        <property name="editTriggers" >
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="alternatingRowColors" >
         <bool>true</bool>
        </property>
        <property name="selectionMode" >
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
        <property name="selectionBehavior" >
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="sortingEnabled" >
         <bool>true</bool>
        </property>
        <property name="columnCount" >
         <number>4</number>
        </property>
        <column>
         <property name="text" >
          <string>Jid</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>OTR</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Save</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Expire</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblStatus" >
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
